Architect-Coders/MU8
# Mu8/santiago/issue4 create domain module

**Create Domain Module:** Issue #4 .

- Add create domain module.

**Testing Done:**

* On real device: Huawei 20P Lite.

**How to test:**

*Create dummy class in the domain module
*Add implementation project (": domain") in the gradle app.
*In the App module (presentation) of the MainActivity.kt class add an instance of the Domain dummy class and verify that the class is imported correctly.

**Signed-off-by:**

* [SalosV](https://github.com/SalosV) + santiagolozada95@gmail.com
